pkg/configtx/consortiums.go

```diff
@@ -244,3 +244,21 @@ func implicitMetaAnyPolicy(policyName string) (*standardConfigPolicy, error) {
 		value: implicitMetaPolicy,
 	}, nil
 }
+
+// getConsortiumOrg returns the organization config group for a consortium org in the
+// provided config. It returns an error if a named consortium or a consortium org was not found
+// in the config with the specified name.
```

Comment:
We've actually decided to no longer return an error if the consortium or consortium org is not found. This will lead to panicking if they don't exist but we figured that behavior was fine for the purposes of easily chaining API calls together as part of [FAB-17744](https://jira.hyperledger.org/browse/FAB-17744). Instead we've been adding a node to the function comments to indicate that it will panic if not provided.

Reply:
I have removed the error info for function getApplicationOrg, getOrdererOrg, and getConsortiumOrg.